setuptools/msvc.py

```diff
@@ -17,6 +17,10 @@
 
 from more_itertools import unique_everseen
 
+from setuptools.compat import py310
+
+from ._path import StrPath
```

Comment:
Let's use relative imports to align with `from ._path import StrPath`

```suggestion
from ._path import StrPath
from .compat import py310
```